public/login.html

```diff
@@ -80,3 +81,18 @@ <h1>Login</h1>
 
   <button type="submit">Login</button>
 </form>
+
+<script>
+  function login(e) {
+    e.target.action = 'https://api.dev.core.ac.uk/login_check?continue=' + encodeURIComponent(
+      window.location.origin
+    );
+  }
+
+  const urlParams = new URLSearchParams(window.location.search);
+  if (urlParams.has('reason')) {
+    const message = document.getElementById('error-message')
```

Comment:
I am considering using more old JS but whatever works. I am not sure about our browsers list yet. We actually support only latest Chrome but we will need to wide this up.

Maybe `var` and avoiding `URLSearchParams` in favour of pure regular expression test would work better.